homework5/task1.js

```diff
@@ -0,0 +1,126 @@
+const SarahMay = {
+  firstName: "Sarah",
+  lastName: "May",
+  birthDate: "13.03.1990",
+  address: {
+      country: "Spain",
+      city: "Seville",
+      street: "calle Verde",
+      house: "5",
+      apartment: "2A",
+  },
+  job: {
+      title: "teacher",
+      experience: 5
+  }
+}
+
+const JeremieBrown = {
+  firstName: "Jeremie",
+  lastName: "Brown",
+  birthDate: "10.11.1983",
+  address: {
+    country: "Great Britain",
+    city: "Bristol",
+    street: "Park street",
+    house: "14"
+  },
+  job: {
+    title: "sale manager",
+    experience: 10
+  }
+}
+
+const Elizabeth = {
+  firstName: "Elizabeth",
+  birthDate: "27.02.1998",
+  address: {
+    country: "Italy",
+    city: "Verona",
+    street: "via Calchirelli",
+    house: "9F",
+    apartment: "49",
+  },
+  job: {
+    title: "fitness coach",
+    experience: 1.5
+  }
+}
+
+
+//created a constructor
+function PersonFunc (object){
+  this.object = object;
+}
+
+//created a new instance of the object from the constructor
+let person1 = new PersonFunc(SarahMay);
+let person2 = new PersonFunc(JeremieBrown);
+let person3 = new PersonFunc(Elizabeth);
+
+//add methods to the constructor prototype
+PersonFunc.prototype.getFullName = function() {
+  try {
+      if (this.object.firstName && this.object.lastName) {
+        console.log(`${this.object.firstName} ${this.object.lastName}`); 
```

Comment:
Don't need to use the console inside the function,
But You can use the console outside the function